src/components/input/input.stories.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState, useEffect } from "react"
+import { storiesOf } from "@storybook/react"
+import styled from "styled-components"
+import { text, number, boolean } from "@storybook/addon-knobs"
+import { useCallback } from "@storybook/addons"
+import { getColor } from "../../theme/utils"
+import { readmeCleanup } from "../../../utils/readme"
+import { TextInput, useInputValue, useTouchedState } from "."
+import { Icon } from "../icon"
+// @ts-ignore
+import readme from "./README.md"
+
+const inputStory = storiesOf("TextInput", module)
```

Comment:
I have organised the storybook components into categories. (Components, Utils and Layout for the moments). So maybe this line would be good to be set as: `storiesOf("COMPONENTS|Controls/TextInput", module)`

Reply:
I suggest to merge as is (master doesn't contain the change you made yet), and apply the structure later as patch.